specification/conformance/conformance_rules/columns/pricingcurrency.json

```
@@ -0,0 +1,128 @@
+{
+  "PricingCurrency-C-000-C": {
+    "Function": "Composite",
+    "Reference": "PricingCurrency",
+    "EntityType": "Column",
+    "Notes": "",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "ApplicabilityCriteria": [
+      "PRICING_BILLING_CURRENCY_DIFFERENCES_SUPPORTED"
+    ],
+    "Type": "Static",
+    "ValidationCriteria": {
+      "MustSatisfy": "The PricingCurrency column MUST adhere to the following requirements.",
```

nit: Aligning with the verbiage used in the remaining 56 columns (and in the 1.2 column definitions).

```suggestion
      "MustSatisfy": "The PricingCurrency column adheres to the following requirements:",
```